core/src/components/textarea/test/slot/index.html

```diff
@@ -91,8 +91,49 @@ <h2>Outline / Floating</h2>
               <div slot="label">Email <span class="required">*</span></div>
             </ion-textarea>
           </div>
+
+          <div class="grid-item">
+            <h2>Outline / Floating / Async</h2>
+            <ion-textarea id="solid-async" label-placement="floating" fill="outline" value="hi@ionic.io"></ion-textarea>
+          </div>
         </div>
+
+        <ion-button onclick="addSlot()">Add Slotted Content</ion-button>
+        <ion-button onclick="updateSlot()">Update Slotted Content</ion-button>
+        <ion-button onclick="removeSlot()">Remove Slotted Content</ion-button>
```

Comment:
The notch is still present when the slot is removed:

![slot removed](https://i.gyazo.com/687526e6b1a8e7b0eacec933d2bb06cd.png)

Reply:
Ah yeah same issue as your screenshot comment -- I think I need to rebase since I fixed a few bugs related to this in the input PR.